keyboards/xw60/xw60.h

```diff
@@ -0,0 +1,62 @@
+#ifndef XW60_H
```

Comment:
```suggestion
#pragma once
```

Current QMK convention is to use `#pragma once` instead of these include guards.